xds/src/main/java/io/grpc/xds/WrrLocalityLoadBalancer.java

```diff
@@ -107,10 +108,6 @@ public boolean acceptResolvedAddresses(ResolvedAddresses resolvedAddresses) {
               wrrLocalityConfig.childPolicy));
     }
 
-    // Remove the locality weight attribute now that we have consumed it. This is done simply for
-    // ease of debugging for the unsupported (and unlikely) scenario where WrrLocalityConfig has
-    // another wrr_locality as the child policy. The missing locality weight attribute would make
-    // the child wrr_locality fail early.
     resolvedAddresses = resolvedAddresses.toBuilder()
         .setAttributes(resolvedAddresses.getAttributes().toBuilder()
```

Comment:
I actually meant that L111-L113 can be removed, no?

Reply:
Ok that was silly of me. Yes, not needed, I removed the lines.